ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -396,7 +396,11 @@ fi
 # Benewake TFMini
 if param greater SENS_EN_TFMINI 0
 then
-	tfmini start
+	if ver hwcmp PX4FMU_V3
```

Comment:
Why is this specific to this board? Previously we enabled this on every board - since they have to set SENS_EN_TFMINI  for this to be started, what difference does it make?

Reply:
The difference is the device (`-d /dev/ttyS6`). Previously, It could happen that it breaks other devices/sensors...